Pravega: NullPointerException & Timeout in PravegaGraphSpec

```
[info] PravegaGraphSpec:
[info] Pravega connector
[ERROR] [03/10/2021 16:49:48.064] [PravegaSpec-akka.actor.default-blocking-io-dispatcher-9] [PravegaSourcesStageLogic(akka://PravegaSpec)] UNIMPLEMENTED: Method not found: io.pravega.controller.stream.api.grpc.v1.ControllerService/createReaderGroup
[ERROR] [03/10/2021 16:49:48.066] [PravegaSpec-akka.actor.default-blocking-io-dispatcher-9] [akka://PravegaSpec/system/Materializers/StreamSupervisor-0/flow-2-2-PravegaSource] Error during postStop in [akka.stream.alpakka.pravega.impl.PravegaSource@4e30d393]: null
java.lang.NullPointerException
	at akka.stream.alpakka.pravega.impl.PravegaSourcesStageLogic.postStop(PravegaSource.scala:72)
	at akka.stream.impl.fusing.GraphInterpreter.finalizeStage(GraphInterpreter.scala:599)
	at akka.stream.impl.fusing.GraphInterpreter.afterStageHasRun(GraphInterpreter.scala:579)
	at akka.stream.impl.fusing.GraphInterpreter.init(GraphInterpreter.scala:312)
	at akka.stream.impl.fusing.GraphInterpreterShell.init(ActorGraphInterpreter.scala:594)
	at akka.stream.impl.fusing.ActorGraphInterpreter.tryInit(ActorGraphInterpreter.scala:702)
	at akka.stream.impl.fusing.ActorGraphInterpreter.preStart(ActorGraphInterpreter.scala:751)
	at akka.actor.Actor.aroundPreStart(Actor.scala:548)
	at akka.actor.Actor.aroundPreStart$(Actor.scala:548)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundPreStart(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.create(ActorCell.scala:641)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:513)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:535)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:295)
	at akka.dispatch.Mailbox.run(Mailbox.scala:230)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[info] - should runs sources *** FAILED *** (13 seconds, 853 milliseconds)
[info]   java.util.concurrent.TimeoutException: Futures timed out after [10 seconds]
[info]   at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:259)
[info]   at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:187)
[info]   at scala.concurrent.Await$.$anonfun$ready$1(package.scala:195)
[info]   at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:57)
[info]   at scala.concurrent.Await$.ready(package.scala:146)
[info]   at akka.stream.alpakka.pravega.PravegaGraphSpec.$anonfun$new$2(PravegaGraphSpec.scala:79)
[info]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.wordspec.AnyWordSpecLike$$anon$3.apply(AnyWordSpecLike.scala:1076)
[info]   at org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
[info]   at org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
[info]   at org.scalatest.wordspec.AnyWordSpec.withFixture(AnyWordSpec.scala:1879)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.invokeWithFixture$1(AnyWordSpecLike.scala:1074)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$runTest$1(AnyWordSpecLike.scala:1086)
[info]   at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.runTest(AnyWordSpecLike.scala:1086)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.runTest$(AnyWordSpecLike.scala:1068)
[info]   at akka.stream.alpakka.pravega.PravegaGraphSpec.akka$stream$alpakka$testkit$scaladsl$Repeated$$super$runTest(PravegaGraphSpec.scala:17)
[info]   at akka.stream.alpakka.testkit.scaladsl.Repeated.run0$1(Repeated.scala:24)
[info]   at akka.stream.alpakka.testkit.scaladsl.Repeated.runTest(Repeated.scala:28)
[info]   at akka.stream.alpakka.testkit.scaladsl.Repeated.runTest$(Repeated.scala:22)
[info]   at akka.stream.alpakka.pravega.PravegaGraphSpec.runTest(PravegaGraphSpec.scala:17)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$runTests$1(AnyWordSpecLike.scala:1145)
[info]   at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:413)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]   at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
[info]   at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]   at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:396)
[info]   at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:475)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.runTests(AnyWordSpecLike.scala:1145)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.runTests$(AnyWordSpecLike.scala:1144)
[info]   at org.scalatest.wordspec.AnyWordSpec.runTests(AnyWordSpec.scala:1879)
[info]   at org.scalatest.Suite.run(Suite.scala:1112)
[info]   at org.scalatest.Suite.run$(Suite.scala:1094)
[info]   at org.scalatest.wordspec.AnyWordSpec.org$scalatest$wordspec$AnyWordSpecLike$$super$run(AnyWordSpec.scala:1879)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$run$1(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:535)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run$(AnyWordSpecLike.scala:1188)
[info]   at akka.stream.alpakka.pravega.PravegaBaseSpec.org$scalatest$BeforeAndAfterAll$$super$run(PravegaBaseSpec.scala:17)
[info]   at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:213)
[info]   at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
[info]   at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
[info]   at akka.stream.alpakka.pravega.PravegaBaseSpec.run(PravegaBaseSpec.scala:17)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:318)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:513)
[info]   at sbt.ForkMain$Run.lambda$runTest$1(ForkMain.java:413)
[info]   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[info]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[info]   at java.lang.Thread.run(Thread.java:748)
```
https://travis-ci.com/github/akka/alpakka/jobs/489767954#L580

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -258,7 +258,7 @@ services:
     ports:
       - "3128:3128"
   pravega:
-    image: pravega/pravega:0.8.0
+    image: pravega/pravega:0.9.0
     ports:
       - 9090:9090
       - 12345:12345

